docs/src/main/sphinx/sql/explain.rst

```diff
@@ -353,7 +353,9 @@ JSON Logical plan::
 EXPLAIN (TYPE DISTRIBUTED)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Distributed plan::
+Process the supplied query statement and create a distributed plan in text
+format. The distributed splits the plan into stages, and therefore explicitly shows the data exchange between workers::
```

Comment:
`The distributed splits the plan` -> `The distributed plan splits the logical plan`

Reply:
Doesn't seem to be addressed

Reply:
Done